CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Change log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com) and this project adheres to [Semantic Versioning](http://semver.org).
```

Comment:
I would rather have written
`this project follows the Semantic Versioning code`